### PR TITLE
fix ractor's doc. [ci skip]

### PR DESCRIPTION
**Pull request by ko1**

### CODE REVIEW

File: ractor.c [hunk 1]
```diff
@@ -1935,16 +1935,23 @@ ractor_moved_missing(int argc, VALUE *argv, VALUE self)
  *  ClosedError is a descendant of StopIteration, so the closing of the ractor will break
  *  the loops without propagating the error:
  *
- *     r = Ractor.new { 3.times { puts "Received: " + receive } }
+ *     r = Ractor.new do
```
Comment (ko1) [inline ractor.c:1938]: I expect to use it on `receive` side, not sending side.

File: ractor.rb [hunk 2]
```diff
@@ -1,14 +1,19 @@
 # Ractor is a Actor-model abstraction for Ruby that provides thread-safe parallel execution.
 #
-# To achieve this, ractors severely limit data sharing between different ractors.
-# Unlike threads, ractors can't access each other's data, nor any data through variables of
```
Comment (ko1) [inline ractor.rb:4]: It is too trivial thing to show at the beginning of Ractor doc.

File: ractor.c [hunk 3]
```diff
@@ -1935,16 +1935,23 @@ ractor_moved_missing(int argc, VALUE *argv, VALUE self)
...
+ *       loop do
+ *         msg = receive # raises ClosedError and loop traps it
+ *         puts "Received: #{msg}"
+ *       end
+ *     end
```
Comment (marcandre) [inline ractor.c:1943]: I would put a `puts "loop exited"` here

File: ractor.rb [hunk 4]
```diff
@@ -17,8 +22,8 @@
 # On CRuby (the default implementation), Global Virtual Machine Lock (GVL) is held per ractor, so
 # ractors are performed in parallel without locking each other.
 #
-# Instead of accessing the shared state, the data should be passed to and from ractors via
-# sending and receiving messages, thus making them _actors_ ("Ractor" stands for "Ruby Actor").
+# Instead of accessing the shared state, the objects should be passed to and from ractors via
```
Comment (ko1) [inline ractor.rb:25]: data->object

File: ractor.rb [hunk 5]
```diff
@@ -1,14 +1,19 @@
...
+# Ractor.new can make new Ractor and it will run in parallel.
 #
 #     # The simplest ractor
 #     r = Ractor.new {puts "I am in Ractor!"}
-#     r.take # allow it to finish
+#     r.take # wait it to finish
 #     # here "I am in Ractor!" would be printed
 #
+# Ractors do not share usual objects, so the some kind of thread-safety concerns such as data-race,
+# race-conditions are not avialble on multi-ractor programming.
```
Comment (marcandre) [inline ractor.rb:11]: ^avialble^available

File: ractor.rb [hunk 6]
```diff
@@ -71,14 +76,17 @@
 #
 # It is said that Ractor receives messages via the <em>incoming port</em>, and sends them
 # to the <em>outgoing port</em>. Either one can be disabled with Ractor#close_incoming and
-# Ractor#close_outgoing respectively.
+# Ractor#close_outgoing respectively. If a ractor terminated, its ports will be closed
+# automatically.
```
Comment (ko1) [inline ractor.rb:80]: add automatic closing

File: ractor.rb [hunk 7]
```diff
@@ -71,14 +76,17 @@
...
 #
 # == Shareable and unshareable objects
 #
 # When the object is sent to and from the ractor, it is important to understand whether the
-# object is shareable or not. Shareable objects are basically those which can be used by several
-# threads without compromising thread-safety; e.g. immutable ones. Ractor.shareable? allows to
-# check this, and Ractor.make_shareable tries to make object shareable if it is not.
+# object is shareable or unshareable. Most of objects are unshareable objects.
```
Comment (ko1) [inline ractor.rb:85]: I think it is valuable to state that "most of objects are unshareable" and there are several special shareable objects.

File: ractor.rb [hunk 8]
```diff
@@ -138,8 +146,13 @@
 # Notice that even +inspect+ (and more basic methods like <tt>__id__</tt>) is inaccessible
 # on a moved object.
 #
-# Besides frozen objects, shareable objects are instances of Module, Class and Ractor itself. Modules
-# and classes, though, can not access instance variables from ractors other than main:
+# Besides frozen objects, there are shareable objects. Class and Module objects are shareable so
```
Comment (ko1) [inline ractor.rb:149]: There are 3 types:

* immutable (frozen and refer to sharable objects)
* classes/modules
* special shareable
  * Ractor
  * Lvar/TVar/... (will be added by core or extension)

File: ractor.rb [hunk 9]
```diff
@@ -374,7 +392,8 @@ def self.select(*ractors, yield_value: yield_unspecified = true, move: false)
   #     Still received only one
   #     Received: message2
   #
-  # If close_incoming was called on the ractor, the method raises Ractor::ClosedError:
+  # If close_incoming was called on the ractor, the method raises Ractor::ClosedError
+  # if there is no more messages in incomeing queue:
```
Comment (marcandre) [inline ractor.rb:396]: => if there are no more messages in incoming queue

File: ractor.rb [hunk 10]
```diff
@@ -17,8 +22,8 @@
 # On CRuby (the default implementation), Global Virtual Machine Lock (GVL) is held per ractor, so
 # ractors are performed in parallel without locking each other.
 #
-# Instead of accessing the shared state, the data should be passed to and from ractors via
-# sending and receiving messages, thus making them _actors_ ("Ractor" stands for "Ruby Actor").
```
Comment (ko1) [inline ractor.rb:21]: Message passing is one important thing about actor-model, but I think the most important point is it is isolated with other actors. So I can't say it is the reason why it is an actor.

**Pull request merged**